code/modules/client/client procs.dm

```diff
@@ -271,8 +271,8 @@
 
 	// Change the way they should download resources.
 	if(config.resource_urls)
-		src.preload_rsc = pick(config.resource_urls)
-	else src.preload_rsc = 1 // If config.resource_urls is not set, preload like normal.
+		preload_rsc = pick(config.resource_urls)
+	else preload_rsc = 1 // If config.resource_urls is not set, preload like normal.
```

Comment:
I should point out that this bit of code doesn't actually work. Byond only looks at the compile time value of this var to decide where they download the rsc.